build/yaml/samples/nsx_v1alpha1_vpcnetworkconfigurations.yaml

```diff
@@ -22,4 +22,5 @@ metadata:
 spec:
   vpc: /orgs/default/projects/proj-1/vpcs/vpc-1
   defaultIPv4SubnetSize: 28
```

Comment:
please change defaultSubnetSize: to 32, since subnetszie must be power of 2.
so, 28 is invalid.